src/util/validation.ts

```diff
@@ -1,7 +1,8 @@
 // Shared validation utilities
 
-// UUID v4 regex pattern
-const UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
+// UUID v4 regex pattern, PLUS allowing extra "2" at end for backsides of cards.
+// See convertId in update_cards.ts. Relevant to card IDs, not Oracle ids.
+const UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}2?$/i;
```

Comment:
Allow optional 2 at the end.

I checked in drafting where uuid validation is also used, but in that case it passes the oracle id to the backend so doesn't have the same problem